xwa_hook_tourmultiplayer/zip/hook_tourmultiplayer.cfg

```diff
@@ -16,8 +16,3 @@ ChangeCombatSimMusic = 1
 ; TourOfDutyButtonRequiredMissionIndex = value
 ; Define the index of the mission when the Tour of Duty button become active.
 TourOfDutyButtonRequiredMissionIndex = 1
-
-; GreenAndRedForIFFColorsOnly = 0 (default) or 1
-; When set to 0, use IFF colors
-; When set to 1, change IFF colors to green for friendlies and red for enemies
-GreenAndRedForIFFColorsOnly = 0
```

Comment:
Why remove this setting?

Reply:
I'm not sure how that happened. Something must have gotten lost during the sync.

Reply:
Ok it's fixed now.